package/harvester-os/files/usr/share/rancher/rancherd/config.yaml.d/50-defaults.yaml

```diff
@@ -3,7 +3,7 @@ rancherValues:
   rancherImage: rancher/rancher
   rancherImageTag: v2.6.11
   noDefaultAdmin: false
-  features: multi-cluster-management=false,multi-cluster-management-agent=false
+  features: multi-cluster-management=true,multi-cluster-management-agent=false
```

Comment:
can u please help to elaborate on what the `multi-cluster-management-agent` is used for and why we don't need it?

Reply:
Added it. Thanks.